test/e2e/suites/e2e/e2e_test.go

```diff
@@ -469,6 +471,29 @@ var _ = Describe("e2e tests [PR-Blocking]", func() {
 				return 1
 			}, e2eCtx.E2EConfig.GetIntervals(specName, "wait-worker-nodes")...).Should(Equal(1))
 			Expect(trunk.PortID).To(Equal(port.ID))
+			shared.Logf("Trunk ID: %s", trunk.ID)
+
+			// Add subports to the trunked port.
+			shared.Logf("Adding subports to the trunked port")
+			providerClient, clientOpts, _, err := shared.GetTenantProviderClient(e2eCtx)
+			Expect(err).To(BeNil(), "Cannot create providerClient")
+			computeClient, err := openstack.NewComputeV2(providerClient, gophercloud.EndpointOpts{
+				Region: clientOpts.RegionName,
+			})
```

Comment:
Not 100% sure, but I think this might be your problem: I'd expect you to need a network client here.